tests/PHPCSVersions.php

```diff
@@ -27,7 +27,7 @@ final class PHPCSVersions
      *
      * @var string
      */
-    const NEXT_MAJOR = '4.0.x-dev as 3.9.99';
+    const NEXT_MAJOR = '4.x-dev as 3.9.99';
```

Comment:
Version 3.10 already exists.
```suggestion
    const NEXT_MAJOR = '4.x-dev as 3.99.99';
```

Reply:
Ouch.. thanks for catching that and yes, 100% agree, that's what it should have read in the first place.